### ISSUE TITLE
Polite/propper way to invite/fork outside projects?

### ISSUE THREAD

**Issue by tomByrer** [regexhq/regexhq#9]
eg https://github.com/gajus/postwoman
Just post an issue on their repo, "Hey join the party over here please!"?
After no response after X time, then fork/copy-paste to here?

**Comment by tunnckoCore** [comment]
Sounds good. :+1:

**Comment by tomByrer** [comment]
I've been adding them to the [WiKi](https://github.com/regexps/meta/wiki#other-regex-validators) also.

**Comment by kevva** [comment]
Why should we create duplicate modules? If they aren't answering it's fine. We don't need to cover everything.

**Comment by jonschlinkert** [comment]
> Why should we create duplicate modules? If they aren't answering it's fine. We don't need to cover everything.

Agreed. Why do this? My own rule of thumb is to only create new repos when existing projects have different goals, or the repos are not maintain or are poorly maintained. Let's just focus on filling gaps and just being useful in general.
